src/global-styles.js

```diff
@@ -12,6 +12,10 @@ const GlobalStyle = createGlobalStyle`
   margin-bottom: 2px !important;
 }
 
+.overflow-wrap {
+  overflow-wrap: anywhere;
```

Comment:
We might potentially have an issue with browser support. I believe Safari (the new IE) does not support the `anywhere` wrap.

We might want to consider adding some fallback with `@supports` for those who still use safari.

Reply:
@Hyperkid123, @epwinchell - I did not see this comment before merging - this can be addressed in a separate PR

Reply:
@Hyperkid123 Will do. Thanks.